src/pixChargeback/pixChargeback.php

```diff
@@ -26,16 +26,17 @@ class PixChargeback extends Resource
 
     ## Parameters (optional):
         - description [string, default null]: description for the PixChargeback.
+        - tags [array of strings, default []]: array of strings for tagging. ex: ["travel", "food"]
 
     ## Attributes (return-only):
         - id [string]: unique id returned when the PixChargeback is created. ex: "5656565656565656"
         - analysis [string]: analysis that led to the result.   
-        - bacenId [string]: central bank's unique UUID that identifies the PixChargeback.
         - senderBankCode [string]: bankCode of the Pix participant that created the PixChargeback. ex: "20018183"
         - receiverBankCode [string]: bankCode of the Pix participant that received the PixChargeback. ex: "20018183"
         - rejectionReason [string]: reason for the rejection of the Pix chargeback. Options: "noBalance", "accountClosed", "unableToReverse"
-        - chargebackId [string]: return id of the chargeback transaction. ex: "D20018183202202030109X3OoBHG74wo".
+        - reversalReferenceId [string]: return id of the reversal transaction. ex: "D20018183202202030109X3OoBHG74wo".
```

Comment:
Tem que trocar essa Docstrings na vdd:
`returnId or endToEndId of the chargeback transaction`

Reply:
check